src/tleextension.c

```diff
@@ -2808,7 +2808,7 @@ PG_FUNCTION_INFO_V1(pg_tle_extension_update_paths);
 Datum
 pg_tle_extension_update_paths(PG_FUNCTION_ARGS)
 {
-	Name		extname = PG_GETARG_NAME(0);
+	char*		extname = text_to_cstring(PG_GETARG_TEXT_P(0));
```

Comment:
To make pg_indent happy:
```suggestion
	char	   *extname = text_to_cstring(PG_GETARG_TEXT_P(0));
```